manim/mobject/logo.py

```diff
@@ -41,10 +45,12 @@ def construct(self):
         class BannerLightBackground(Scene):
             def construct(self):
                 self.camera.background_color = "#ece6e2"
-                banner = ManimBanner(dark_theme=False).scale(0.5).to_corner(UR)
-                self.play(FadeIn(banner))
-                self.play(banner.expand())
-                self.play(FadeOut(banner))
+                banner_large = ManimBanner(dark_theme=False).scale(0.7)
+                banner_small = ManimBanner(dark_theme=False).scale(0.35)
+                banner_small.next_to(banner_large, DOWN)
+                self.play(banner_large.create(), banner_small.create())
+                self.play(banner_large.expand(), banner_small.expand())
```

Comment:
Comment (non-blocking): I now realize that this is one of the only (perhaps the only?) instance in the whole library where a mobject's method returns an animation. This is not Bad per se, but I just want to point that out.

We have discussed similar things in an issue that I can't find anymore (I think @jsonV was involved?): when we were discussing replacing get/set methods with a single `set` method... :thinking:

Reply:
I thought that returning the animation itself is most convenient (also, I wasn't sure whether it is even possible to have an object method actually run the animation -- I would guess not?

I'm happy to adapt to another approach, if you find one that is more suitable -- but I kind of like the fact that this directly returns the animation. :-)

Reply:
No no no, I like it! I guess what I'm saying is we should move more toward this style in the future. But for now, this will be a sticking point since it's the sole class that does it.

Reply:
Found it: [this is the discussion I meant to link to](https://github.com/ManimCommunity/manim/issues/787#issuecomment-737400927).